test/test-debugletapi.js

```diff
@@ -53,8 +53,9 @@ describe('Debuglet API', function() {
   });
 
   it('should acquire the project number during init', function(done) {
-    debugletapi.init('uid123', { warn: function() {} }, function(err) {
+    debugletapi.init('uid123', { warn: function() {} }, function(err, project) {
       assert(!err);
+      assert.equal(debugletapi.project_, project);
```

Comment:
This test will still pass if the project is not set correctly. Maybe we could compare project to the expected value. It looks like it should be `'project123'` from https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/pull/154/files#diff-8cd5c99c65934398b26d2eb9e3bfc2ddL29